From 6858b38e059f4f54079d8f274b88bf723b5f2ce3 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Sat, 19 Oct 2013 13:09:47 +0700
Subject: [PATCH] Use TheSLinux source mirror by default

All source files will be downloaded from #theslinux mirror;
The list of URIs is returned by (_pkgbuild_s_sources).
To use the original PKGBUILD sources, set the variable
WITHOUT_THESLINUX_SOURCES=1
---
 pacman/src/pacman-4.1.0/scripts/makepkg.sh.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
index e75a4d2..66a3049 100644
--- a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
+++ b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
@@ -2780,6 +2780,9 @@ if [[ -n "${_FEATURE_STRING}" && -f "${BUILDFILE}${_FEATURE_STRING}" ]]; then
 	source_safe "${BUILDFILE}${_FEATURE_STRING}"
 fi
 
+[[ -n "${WITHOUT_THESLINUX_SOURCES}" ]] \
+	|| source=($(s-run _pkgbuild_s_sources))
+
 # There are some cases
 #   PACKAGE_BASE is unset:
 #     `pkgname`, `pkgbase` will be from `PKGBUILD`
-- 
1.8.4

